### DISCUSSION TITLE
Does index.redb contain sensitive content?

### DISCUSSION THREAD

**Discussion by dannydeezy** [ordinals/ord#1494] [Q&A] [answered]
does this file have any sensitive info specific to your node/wallets?
what are the security considerations of copying a fully synced `index.redb` from someone else and using that?

**Comment by raphjaph** [comment, marked as answer]
No wallet state is stored in `index.redb`, all sensitive information is stored inside a Bitcoin Core wallet. Copying and sharing `index.redb` should not be a problem.

**Comment by dude513** [comment]
Can someone please send a link to a fully indexed `index.redb` file? It will be super helpful!
It should be somewhere between 65 and 70 megabytes.

**Comment by casey** [comment]
One note: We have not fully thought through the security implications of loading untrusted `index.redb` files. There may be bugs in `ord` or `redb` which make this an exploit vector, so it's not a supported way of syncing an ord node.

**Reply by DarthMalloc** [reply]
Any updates / new insight on potential security implications of loading an index file? (seeing a lot more of this these days)

**Reply by DrJingLee** [reply]
Currently we have some source of **redb** file for download, but it has an unknown risk or potential attacks.
Is it good to give out an **official** version of the redb file for users to download ? Kind of trusted version with MD5 verification etc. I can offer a cloud or paid space .

**Comment by ottosch** [comment]
An user on Discord asked me to upload the index file and I started doing it on this telegram channel: https://t.me/ordinalsIndex
If anyone wanna give it a try, files go from `xaa` to `xap`. On Linux and MacOS, you have to download them all and join them with `cat xa* > index.redb`.

I don't know how to join them on Windows; someone can add instructions for Windows here.
edit: it seems on Windows the command should be `type xa* > index.redb`.

Sha1 of file: a16dccf18b8a65a93942ce3633aed07bfc20580f

This message can be deleted if it violates any rules. Even though I have no idea how I could exploit this, mind @casey's remarks on downloading it.

**Reply by dude513** [reply]
After 2 hours of loading the remaining 100 blocks and one crash it worked, thank you!

**Reply by bliotti** [reply]
just need to warn users, think it is ok to leave up

WARNING POTENTIALLY UNSAFE
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!! https:// t.me/ordinalsIndex !!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Reply by alex27riva** [reply]
> An user on Discord asked me to upload the index file and I started doing it on this telegram channel: https://t.me/ordinalsIndex If anyone wanna give it a try, files go from `xaa` to `xap`. On Linux and MacOS, you have to download them all and join them with `cat xa* > index.redb`.
> 
> I don't know how to join them on Windows; someone can add instructions for Windows here. edit: it seems on Windows the command should be `type xa* > index.redb`.
> 
> Sha1 of file: a16dccf18b8a65a93942ce3633aed07bfc20580f
> 
> This message can be deleted if it violates any rules. Even though I have no idea how I could exploit this, mind @casey's remarks on downloading it.

Please make a torrent for faster download

**Reply by dude513** [reply]
> An user on Discord asked me to upload the index file and I started doing it on this telegram channel: https://t.me/ordinalsIndex If anyone wanna give it a try, files go from `xaa` to `xap`. On Linux and MacOS, you have to download them all and join them with `cat xa* > index.redb`.
> 
> I don't know how to join them on Windows; someone can add instructions for Windows here. edit: it seems on Windows the command should be `type xa* > index.redb`.
> 
> Sha1 of file: a16dccf18b8a65a93942ce3633aed07bfc20580f
> 
> This message can be deleted if it violates any rules. Even though I have no idea how I could exploit this, mind @casey's remarks on downloading it.

Any chance the last portion can be updated to have the index until the most recent block? @ottosch 
The reason is that if someone grabs the index file that is behind more than ~600 blocks then it won't index.
One must keep everything up to date, for new users even your index file won't help.

**Reply by dvncan** [reply]
which version is this index file from? I am trying to index 2 nodes now! 
1. is stuck at 790150/791059 and I am getting the RPC error.
2. is just starting but doesnt work with the provided file^ because this error: error: Manual upgrade required. Expected file format version 109, but file is version 107 

Any help is appreciated, especially the author of this ticket.. Would love to hear how error was solved.

**Comment by nisilag** [comment]
how often should I run the index command to keep it up to date?

**Comment by dvncan** [comment]
can someone please point me to where this file lives in a node so i can swap it?

**Reply by nick07002** [reply]
I have the same question  :(
